examples/python/cloud_cluster.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2023, Crate.io Inc.
+# Distributed under the terms of the Apache 2 license.
+"""
+About
+=====
+
+Example program demonstrating how to manage a CrateDB Cloud database cluster.
+It obtains a database cluster identifier or name, connects to the database
+cluster, optionally deploys it, and runs an example workload.
+
+Synopsis
+========
+::
+
+    # Install utility package.
+    pip install 'cratedb-toolkit'
+
+    # Log in to CrateDB Cloud.
+    croud login --idp azuread
```

Comment:
There is an alternative, headless way of doing that: Using `croud config show`, you can display the location of the croud configuration file in YAML format.

```yaml
current-profile: cratedb.cloud
default-format: table
profiles:
  cratedb.cloud:
    auth-token: xxxxxxxxxx
    endpoint: https://console.cratedb.cloud
    key: REDACTED
    organization-id: null
    region: _any_
    secret: xxxxxxxxxx
```

If you fill in the `key` and `secret` values, operations like `croud clusters list` will start working without further ado, even after logging out again using `croud logout`.

You can obtain an API key on your account page [^1], at the "API keys" section.

[^1]: https://console.cratedb.cloud/account/settings